web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -357,6 +358,8 @@ const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
               </OnBoardingCard>
               <ProjectPageFooter
                 onSave={submitForm}
+                // onNext={() => null} // TODO route to /metadata
```

Comment:
this should be done either in this PR or in your other one: #990

Reply:
also see https://github.com/regen-network/regen-web/pull/990#discussion_r901578620